src/VisualStudio/VisualBasic/Repl/VisualBasicVsInteractiveWindowProvider.vb

```diff
@@ -68,6 +68,14 @@ Namespace Microsoft.VisualStudio.LanguageServices.VisualBasic.Interactive
         Protected Overrides Sub LogSession(key As String, value As String)
             Logger.Log(FunctionId.VisualBasic_Interactive_Window, KeyValueLogMessage.Create(Sub(m) m.Add(key, value)))
         End Sub
+
+        Protected Overrides Sub LogCloseSession(languageBufferCount As Integer)
+            Logger.Log(FunctionId.VisualBasic_Interactive_Window,
```

Comment:
Is this the same as VB, other than the ```FunctionId```?  If so, should we just have an abstract ```FunctionId``` property?